mantidproject/mantid
# Add documentation page for Multi-dataset Fitting

**Description of work.**
As part of #22766 I made some interface docs because none existed.

**To test:**

 - Build the docs.
 - Check them

_No associated issue_

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
